api/DOMRectList.json

```diff
@@ -0,0 +1,226 @@
+{
+  "api": {
+    "DOMRectList": {
+      "__compat": {
+        "spec_url": "https://drafts.fxtf.org/geometry/#DOMRectList",
+        "support": {
+          "chrome": [
+            {
+              "version_added": "61"
+            },
+            {
+              "alternative_name": "ClientRectList",
+              "version_added": "2",
+              "version_removed": "61"
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "61"
+            },
+            {
+              "alternative_name": "ClientRectList",
+              "version_added": "18",
+              "version_removed": "61"
+            }
+          ],
+          "edge": [
+            {
+              "version_added": "79"
+            },
+            {
+              "alternative_name": "ClientRectList",
+              "version_added": "12",
+              "version_removed": "79"
+            }
+          ],
+          "firefox": [
+            {
+              "version_added": "27"
+            },
+            {
+              "alternative_name": "ClientRectList",
+              "version_added": "3",
+              "version_removed": "27"
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "27"
+            },
+            {
+              "alternative_name": "ClientRectList",
+              "version_added": "4",
+              "version_removed": "27"
+            }
+          ],
+          "ie": {
+            "alternative_name": "ClientRectList",
+            "version_added": "9"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/getClientRects#browser_compatibility says IE 5 for `element.getClientRects()`. Looks like in IE 8 it returned a `TextRectangleList`. Let's not bother recording that in BCD :)